pkg/processor/runtime/python/test/python_test.go

```diff
@@ -318,17 +325,21 @@ func (suite *TestSuite) TestNonUTF8Headers() {
 	createFunctionOptions := suite.GetDeployOptions("non-utf8-headers",
 		path.Join(suite.GetTestFunctionsDir(), "common", "empty", "python"))
 	createFunctionOptions.FunctionConfig.Spec.Handler = "empty:handler"
+	internalServerErrorStatus := http.StatusInternalServerError
+	okStatus := http.StatusOK
```

Comment:
A bit awkward, can't be avoided?

Reply:
no, since test function expects a nil pointer represent http status code